rohmu/object_storage/swift.py

```diff
@@ -297,17 +297,24 @@ def copy_file(self, *, source_key, destination_key, metadata=None, **_kwargs):
 
     def store_file_object(self, key, fd, *, cache_control=None, metadata=None, mimetype=None, upload_progress_fn=None):
         metadata = metadata or {}
+        notify_size = [0]
+
+        def upload_progress_fn_wrapper(bytes_written):
+            notify_size[0] = bytes_written
```

Comment:
If this is kept (cf. comment below), it should accumulate the written bytes, not overwrite the previous value.

Reply:
It seems to me that this is called with the value content-length minus number of bytes not yet transferred, unless I misunderstand the logic in `_store_file_contents()`

Reply:
Same as the previous two :+1: